apps/contract-sync-job/src/main.ts

```diff
@@ -83,10 +83,12 @@ async function getContribution(
   govrn: GovrnProtocol,
   data: { tokenId: number },
 ) {
-  const contrs = await govrn.contribution.list({
-    where: { on_chain_id: { equals: data.tokenId } },
-    first: 1,
-  });
+  const contrs = (
+    await govrn.contribution.list({
+      where: { on_chain_id: { equals: data.tokenId } },
+      first: 1,
+    })
+  ).result;
```

Comment:
It may be good to wrap this in a try catch. I think result will error if list returns an empty